lib/server/cms-data.ts

```diff
@@ -10,3 +10,45 @@ export async function getCmsData(): Promise<CmsData> {
   ).json();
   return jsonData;
 }
+
+export async function getGuideHomePage() {
+  const jsonData = await (
+    await fetch(
+      `https://simplejsoncms.com/api/${process.env.GUIDES_SIMPLEJSONCMS_ID}`,
+    )
+  ).json();
+  const banners = Object.entries(jsonData["home"]["banners"]).map(
```

Comment:
after looking at this code more i'm not sure why we are flattening this just to include the `key` in the `data`. just put the value into the data in the CMS and then we can skip this entirely.